AntiSpam/User.py

```diff
@@ -153,7 +156,11 @@ def propagate(self, value: discord.Message):
         """
         if not isinstance(value, discord.Message):
             raise ValueError("Expected message of type: discord.Message")
-
+        
+        # Just for error handling because this check should be check before calling this method
```

Comment:
Can you expand on your comment with the implementation your thinking of?

Reply:
Yeah I did and after reading it myself idk why I wrote it that way, should be better explained now